TLM/CSUtil.Commons/Log.cs

```diff
@@ -59,7 +59,8 @@ private enum LogLevel {
                     File.Delete(LogFilename);
                 }
             }
-            catch (Exception) { }
+            catch (Exception) {
```

Comment:
Is there even need for the `Exception` here?

Reply:
I only wanted to fix the warning, not change behavior.
Personally I think such global "Ignore all exceptions" blocks are a no go.

Reply:
I guess in this instance it would be pointless to try logging an exception (: Unless it was pumped to `Debug.Log()`, which might also fail.

Reply:
Either pointless, or it would destroy the previous logfile, or it would just defer the exception until LogToFile is called. We actually have no clue, since we don't know what caused it. We told it to ignore anything that can happen, while not handling anything at all.

Reply:
Well, if our log file is failing for some reason, we can't use it. So only alternative is Unity `Debug.Log()`; at least then we have some chance of learning about the error.

Reply:
ohh, you are talking about mitigations?
yeah sure debug.log would be a valid strategy. i m just saying that hiding errorsnunder the carpet does not fix them. in that case i would always default to letting the exeption bubble up instead. everything else just causes more pain in the end, at peast that is my experience.

Reply:
Ok, can we `Debug.Log()` the exception then? I know it's not specifically related to this PR but it doesn't really merit a separate PR.